controllers/image/harbor_v1.10.0.go

```diff
@@ -0,0 +1,50 @@
+package image
+
+type harborV1_10_0_ImageLocator struct {
```

Comment:
It seems `V1_10_0_` does not match golang name convention.

Reply:
I'm thinking if we can find a way to simplify the ImageLocator implementation.

 We have component object and this object can have a name such as `harbor-core`, and also we have spec.version, then most image paths can be determined by repo namespace `goharbor`, component.name and spec.version. v := fmt.Sprintf("%s/%s"%s", "goharbor",component.name, spec.version)。 For other 3rd-party images, use current directly returning way.

Reply:
Although `V1_10_0_` may not match golang name convention, but I think it is more clear.